packages/python-packages/apiview-copilot/scratch/output/python/keyvault_secrets_4.10.0b1.json

```diff
@@ -0,0 +1,48 @@
+{
+    "comments": [
+        {
+            "rule_ids": [],
+            "line_no": 1,
+            "bad_code": "",
+            "suggestion": null,
+            "comment": "Here is a summary of the service described by this APIView:\n\nService Overview:  \nThis API provides operations for managing secrets within the Azure Key Vault. It supports functionalities such as setting, retrieving, backing up, deleting, recovering, purging, and listing secrets along with their properties. Both synchronous and asynchronous client implementations are provided to accommodate different usage scenarios.\n\nAPI Version:  \nThe API version is represented by the `ApiVersion` object. The latest version is V7_6_PREVIEW_2.\n\nClient Classes:  \nThe primary client class is the synchronous `SecretClient`, which exposes the following methods: `backup_secret`, `begin_delete_secret`, `begin_recover_deleted_secret`, `close`, `get_deleted_secret`, `get_secret`, `list_deleted_secrets`, `list_properties_of_secret_versions`, `list_properties_of_secrets`, `purge_deleted_secret`, `restore_secret_backup`, `send_request`, `set_secret`, and `update_secret_properties`. The asynchronous counterpart, also named `SecretClient` within the asynchronous namespace, supports methods: `backup_secret`, `close`, `delete_secret`, `get_deleted_secret`, `get_secret`, `list_deleted_secrets`, `list_properties_of_secret_versions`, `list_properties_of_secrets`, `purge_deleted_secret`, `recover_deleted_secret`, `restore_secret_backup`, `send_request`, `set_secret`, and `update_secret_properties`.\n\nOther Classes:  \nThe API also defines several domain models including `DeletedSecret`, `KeyVaultSecret`, and `KeyVaultSecretIdentifier` for managing secret states and identities, as well as `SecretProperties` which encapsulates the attributes of a secret. These classes provide initializers and string representations to support the overall data structure of the service.",
+            "source": "summary"
+        },
+        {
+            "rule_ids": [],
+            "line_no": 4,
+            "bad_code": "namespace azure.keyvault.secrets",
```

Comment:
Hm these namespace comments seem to keep escaping the filter

Reply:
Can tackle in a separate PR if it keeps being problematic.